src/wedge.rs

```diff
@@ -61,6 +61,8 @@ impl Widget<bool> for Wedge {
     }
 
     fn paint(&mut self, ctx: &mut PaintCtx, expanded: &bool, env: &Env) {
+        let size = env.get(theme::BASIC_WIDGET_HEIGHT);
+        let y_offset = ((size - 8.0) / 2.0).floor();
```

Comment:
you should use `ctx.size()` because parent might force some other size

Reply:
Thanks, updated the PR.